src/extension/sldService/src/test/java/org/geoserver/sldservice/rest/ClassifierTest.java

```diff
@@ -1823,8 +1823,8 @@ public void testClassifyRasterSingleFloat() throws Exception {
         assertEquals(2, entries.length);
         ColorMapEntry cm0 = cm.getColorMapEntry(0);
         assertThat(cm0.getQuantity().evaluate(null, Float.class), Matchers.lessThanOrEqualTo(10f));
-        assertEquals("#FF071C", cm0.getColor().evaluate(null, String.class));
-        assertEquals(1, cm0.getOpacity().evaluate(null, Double.class), 0);
+        assertEquals("#000000", cm0.getColor().evaluate(null, String.class));
```

Comment:
It would be useful to describe what this test is testing. You changed the test because the previous one was testing the actual code  behaviour, not the expected behaviour.
What I would do is explaining better what we are testing here, either using comments or refactoring the tests to make the intent more evident. It would be useful to do this for changed tests and new ones.

Reply:
fixed